### ISSUE TITLE
[BUG] mcbserver install and update failing

### ISSUE THREAD

**Issue by us3r1d** [GameServerManagers/LinuxGSM#3505]
## User Story

`./mcbserver install` failed on "Checking remote build"; I checked an existing server with `./mcbserver update` and it failed at the same point.

## Basic info

* **Distro:** CentOS 8
* **Game:** Minecraft Bedrock
* **Command:** mcbserver
* **LinuxGSM version:** v21.2.4

## Further Information

This appears to be caused by Minecraft changing their web server to block access from scriptable command-line clients (see the Mojang bug at <https://bugs.mojang.com/browse/WEB-4753>).  Changing the curl command in the install and update scripts from:

curl -Ls "https://www.minecraft.net/en-us/download/server/bedrock/"

to:

/usr/bin/curl --cookie -Ls -H "user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.182 Safari/537.36" "https://www.minecraft.net/en-us/download/server/bedrock"

got updates working again and allowed install to complete, though `./mcbserver start` still doesn't work so there's probably something else as well; I'll update when/if I get that working.

(I don't use curl much, so that command was put together by referencing the man pages and web searching; I have no doubt it can be done better.)

**Comment by us3r1d** [comment]
The issue preventing start from working was a missing package that the instructions and prereq check didn't include so I opened a new bug for it at <https://github.com/GameServerManagers/LinuxGSM/issues/3506>.

**Comment by dgibbs64** [comment]
Thanks to reporting this. This will be a priority bug fix.

**Comment by dgibbs64** [comment]
Just checked this and still working on Ubuntu 20.04 currently. CentOS 8 will need extra work that will be linked with your other PR. The new API end point will be used in a future release.

**Comment by pastorhudson** [comment]
I’m also unable to install mcbserver on fresh server install of Ubuntu 20.04 LTS. Valheim installed fine. But mcbserver install fails as @us3r1d describes. I’m happy to provide logs if needed.

**Comment by Ir0nsh007er** [comment]
Same bug on Debian 10

**Comment by asgarov** [comment]
Ubuntu 20.04.2 LTS
LinuxGSM v21.2.4

I have the same problem updating the server. Tried both suggestions above (the one in the description and https://github.com/GameServerManagers/LinuxGSM/issues/3505#issuecomment-876459810), neither helped, although there is a difference in the output:

Before the change:

```
./mcbserver update
[ FAIL ] Updating mcbserver: Checking remote build: minecraft.net
```

After the change:
```
./mcbserver update
[  OK  ] Updating mcbserver: Checking local build: minecraft.net  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   312  100   312    0     0   9176      0 --:--:-- --:--:-- --:--:--  9176
[ FAIL ] Updating mcbserver: Checking remote build: minecraft.net
```

**Comment by Ir0nsh007er** [comment]
wow ty work fine

**Comment by h3o66** [comment]
Fixed with version v21.2.4
